system/lib/pthread/library_pthread_stub.cpp

```diff
@@ -0,0 +1,31 @@
+#include <stdlib.h>
+#include <pthread.h>
+
+extern "C" {
+
+int pthread_key_create(pthread_key_t *key, void (*destructor)(void *))
+{
+	void **ptr = (void**)malloc(sizeof(void*));
+	*ptr = NULL;
+	*key = (pthread_key_t)ptr;
+	return 0;
+}
+
+int pthread_key_delete(pthread_key_t key)
+{
+	free((void*)key);
+	return 0;
+}
+
+void *pthread_getspecific(pthread_key_t key)
+{
+	return *(void**)key;
```

Comment:
This should not be fake, but should work ok. Notice that I use the key as the pointer to the memory, and this line dereferences the key. Or I wonder if I botched something here.